components/core/Server.py

```diff
@@ -44,4 +44,7 @@ def serve_ui1():
 
 # Import routes
 from routes import *
-from file_sender import *
\ No newline at end of file
+from file_sender import sender
+
+# register endpoints
+server.add_url_rule('/api/get/file', 'send_file_from_path', sender.send_file_from_path, methods=['GET'])
```

Comment:
Why do we register the endpoint here instead of REST.py?

Reply:
Oh! Where have we defined the rest of  `/api/*` endpoints in the current version?

Reply:
It's in routes 😊

Reply:
Great. In that case, let's have this in routes/Downloads.py. In the meantime let's change endpoint from `api/get/file` to `api/file` (or something suitable) as well. Usually, endpoints should consist of nouns. https://restfulapi.net/resource-naming/

Reply:
Ok, I have added new commit which have endpoint name `api/file` and endpoint is registered in Download.py